src/RobotInterface/src/Helper.cpp

```diff
@@ -17,12 +17,6 @@ bool RobotInterface::getWorstError(const iDynTree::VectorDynSize& desiredJointPo
         return false;
     }
 
-    if(!m_encodersInterface->getEncoders(m_positionFeedbackDeg.data()))
-    {
-        yError() << "[RobotInterface::getWorstError] Error reading encoders.";
-        return false;
-    }
-
```

Comment:
Probably It was a mistake! Maybe we removed that for the humanoids paper because there was some errors int the joint position in torque control push recovery!

Reply:
I would try to restore it then :thinking:

Reply:
I restored that in previous comments.